packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -190,6 +191,19 @@ export class QueryExportState {
     );
   }
 
+  get unallowedPersistInfo(): string | undefined {
+    if (this.persistQueryState.isInProgress) {
```

Comment:
I don't think this is the right and exhaustive validation checks, let's simplify, remove this and just use `allowPersist`